plugins/env/app/controllers/environment_variable_groups_controller.rb

```diff
@@ -54,7 +54,10 @@ def preview
       @project = Project.find(params[:project_id])
     end
 
-    @groups = SamsonEnv.env_groups(Deploy.new(project: @project), deploy_groups, preview: true)
+    options = {project_specific: params[:project_specific]}
+    options.merge!(params[:preview] ? {preview: true, resolve_secrets: true} : {resolve_secrets: false})
```

Comment:
before it used `preview` mode when preview was not given ... so to not break that, we have to add a new case for `preview == false` (boolean or string supported) ?
```suggestion
    options.merge!(params[:preview].to_s == "false" ? {resolve_secrets: false} : {preview: true, resolve_secrets: true})
```